### DISCUSSION TITLE
how to enable reading mode?

### DISCUSSION THREAD

**Discussion by superiums** [qutebrowser/qutebrowser#8211] [Q&A] [answered]
it's common in other browser, but how to turn it on in qute?

**Comment by The-Compiler** [comment, marked as answer]
See the [readability-js userscript](https://github.com/qutebrowser/qutebrowser/blob/main/misc/userscripts/readability-js). The comment at the start of the file explains how to set things up. After that, you should be able to spawn it directly via `:spawn --userscript readability-js` if your distribution ships it with qutebrowser. If not, download the file and use the full path to it instead.
